lib/msal-node/README.md

```diff
@@ -82,7 +82,7 @@ Any major MSAL Node release:
 
 | MSAL Node version | MSAL support status     | Supported Node versions |
 |-------------------|-------------------------|-------------------------|
-| 1.x.x             | Active development      | 10, 12, 14              |
+| 1.x.x             | Active development      | 10, 12, 14, 15          |
```

Comment:
I am still against formally supporting Node 15, given it is technically unstable and will be EOL in 3 months. I think apps wanting to support it should have to bypass the engines check.

Reply:
I am convinced. Reverted these changes.